.github/boring-cyborg.yml

```diff
@@ -225,9 +225,9 @@ labelPRBasedOnFilePath:
     - providers/tests/system/docker/**/*
 
   provider:edge:
-    - providers/src/airflow/providers/edge/**/*
-    - docs/apache-airflow-providers-edge/**/*
-    - providers/tests/edge/**/*
+    - providers/edge/src/airflow/providers/edge/**/*
```

Comment:
Actually found it out when working on celery. This one could be simplified to just:

  - providers/edge/**

Reply:
That's actually the `gist` of the move of providers to separate sub-directory :)